magicctapipe/io/io.py

```diff
@@ -990,17 +988,19 @@ def load_mc_dl2_data_file(input_file, quality_cuts, event_type, weight_type_dl2)
 
     if viewcone_diff < u.Quantity(0.001, unit="deg"):
         # Handle ring-wobble MCs as same as point-like MCs
-        viewcone = 0 * u.deg
-    else:
-        viewcone = max_viewcone_radius
+        # viewcone = 0 * u.deg
+        min_viewcone_radius = max_viewcone_radius
+    # else:
+    # viewcone = max_viewcone_radius
```

Comment:
the commented out parts about the viewcone can be probably removed

Reply:
Done.